README.md

```diff
@@ -11,6 +11,32 @@
 
 🐺 φεγγάρι - The Lua VM written in JS ES6 for Node and the browser
 
+This repository contains the core fengari code (which is a port of the Lua C library) which includes parser, virtual machine and base libraries.
+However it is rare to use this repository directly.
+
+- To use fengari in a web browser as easily as you might use JavaScript, see [fengari-web](https://github.com/fengari-lua/fengari-web)
+- [fengari-interop](https://github.com/fengari-lua/fengari-interop) is a lua library that makes interoperating with JavaScript objects simple, it is already included in fengari-web.
+- For a clone of the `lua` command line tool, but running under node.js, see [fengari-node-cli](https://github.com/fengari-lua/fengari-node-cli)
+
+### The JS API
+
+Once you've loaded fengari, you can use the JS API:
```

Comment:
s/loaded/required ?

Reply:
well loaded is more generic, if you used a `<script>` you didn't do `require('fengari-web')`